cmd/conformance/main.go

```diff
@@ -237,7 +237,7 @@ func main() {
 
 		// Check bundle and trusted root for Tlog information
 		if len(tr.TlogAuthorities()) > 0 && b.HasInclusionPromise() {
-			verifierConfig = append(verifierConfig, verify.WithTransparencyLog(1))
+			verifierConfig = append(verifierConfig, verify.WithTransparencyLog(1), verify.WithObserverTimestamps(1))
```

Comment:
Minor: In trying to make sure I understand this pull request, I'm wondering if this should be `verify.WithIntegratedTimestamps(1)` instead.

Generally, I think we want people to use `WithObserverTimestamps()` and not have to reason about the specifics of `WithIntegratedTimestamps()` or `WithSignedTimestamps()`.

But I thought what we were saying on https://github.com/sigstore/sigstore-go/pull/42 is that the conformance test will infer the verification policy from the trust root, and if the trust root has `tlogs` entries, we should rely on them both for inclusion and for establishing a timestamp?

I guess when inferring the policy from the trust root, there isn't a way to differentiate today. This isn't a problem really, but it does highlight the complexity of the problem space generally!

Reply:
You're right, this should be `WithIntegratedTimestamps` since this is inferring a policy based on if a promise is present. I'll update in a follow up.

In order to infer `WithObserverTimestamps`, which would be more permissive, it would probably look something like a check for either bundled timestamps or SETs. We could do that here, but we can be more precise with `WithSignedTimestamps` and `WithIntegratedTimestamps`.